java/tsfile/src/main/java/org/apache/tsfile/read/common/block/TsBlock.java

```diff
@@ -213,6 +213,12 @@ public Column[] getColumns(int[] columnIndexes) {
     return columns;
   }
 
+  public Column[] getAllColumns() {
```

Comment:
add java doc for this method, specially for its return value

Reply:
Done.